docs/pages/docs/core-concepts/caching.mdx

```diff
@@ -192,11 +192,13 @@ You can control `turbo`'s caching behavior based on
 the values of environment variables:
 
 - Including environment variables in the `env` key in your `pipeline` definition will impact the cache fingerprint on a per-task or per-workspace-task basis.
+- Including any gitignored environment files that are read by tasks in `globalDependencies`.
```

Comment:
```suggestion
- Including any git-ignored environment files that are read by tasks in `globalDependencies`.
```

Reply:
I tried to match the format used here, although `git-ignored` does make more sense to me.

<img width="558" alt="Screen Shot 2022-11-17 at 4 53 54 PM" src="https://user-images.githubusercontent.com/1154586/202584789-c01a5edb-ee31-42df-b163-66b603448927.png">

Reply:
ah, feel free to ignore my suggestion then :). I thought we were using hyphenated in other places, but maybe I was remembering `non-gitignored`

Reply:
Oh, I just changed it to `non-git-ignored` 😆 Should I change it back?

Reply:
The file is `.gitignore` so I could see arguments for both 🤷